distributed/tests/test_utils_test.py

```diff
@@ -16,6 +16,7 @@
 
 import pytest
 import yaml
+from _pytest.outcomes import Failed
```

Comment:
Using private classes seems a little dangerous.

Reply:
It's made public in `pytest.fail.Exception`

Reply:
I didn't like that either. Cool!